interface/super/edit_layout.php

```diff
@@ -777,35 +778,15 @@ function writeFieldLine($linedata)
     echo "<input type='text' name='fld[$fld_line_no][datacols]' value='" .
          htmlspecialchars($linedata['datacols'], ENT_QUOTES) . "' size='3' maxlength='10' class='optin' style='width:100%' />";
     echo "</td>\n";
-  
-    echo "  <td align='center' class='optcell' style='width:4%' title='" .
-          "A = " . xla('Age') .
-        ", B = " . xla('Gestational Age') .
-        ", C = " . xla('Capitalize') .
-        ", D = " . xla('Dup Check') .
-        ", E = " . xla('Dup Check on only Edit') .
-        ", W = " . xla('Dup Check on only New') .
-        ", G = " . xla('Graphable') .
-        ", I = " . xla('Initially Open Group') .
-        ", L = " . xla('Lab Order') .
-        ", N = " . xla('New Patient Form') .
-        ", O = " . xla('Order Processor') .
-        ", P = " . xla('Default to previous value') .
-        ", R = " . xla('Distributor') .
-        ", T = " . xla('Description is default text') .
-        ", U = " . xla('Capitalize all') .
-        ", V = " . xla('Vendor') .
-        ", X = " . xla('Do Not Print') .
-        ", 0 = " . xla('Read Only') .
-        ", 1 = " . xla('Write Once') .
-        ", 2 = " . xla('Billing Code Descriptions') .
-
-        "'>";
+    // check existing format @TODO for arabic hebrew etc I think mb_strlen is server config specific..check
+    if (mb_strlen($linedata['edit_options'], 'utf8') > 1 && strpos($linedata['edit_options'], ',') === false) {
```

Comment:
Why need to use mb_strlen here? Aren't these always latin chars(A-X and 0-2)? If just looking for it being more than 1 char wouldn't just strlen() suffice here?
(and if do use md_strlen here or somewhere in future, wouldn't it be better to not have a second parameter and default to the local encoding?)

Reply:
Languages like Arabic an Hebrew use symbols and strlen counts bytes thus they should be utf8 right? A character in arabic and the same character in english have different byte counts thus mb. I have to yield to you guys with more experience with translations for guidance here...

Reply:
Ignore my comment as just realized this is not user input. Will change...